app/src/main/java/com/duckduckgo/app/global/api/AtpUnprotectedAppsBucketPixelRemovalInterceptor.kt

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2022 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.app.global.api
+
+import androidx.annotation.VisibleForTesting
+import com.duckduckgo.app.global.plugins.pixel.PixelInterceptorPlugin
+import com.duckduckgo.app.statistics.pixels.Pixel
+import com.duckduckgo.di.scopes.AppScope
+import com.squareup.anvil.annotations.ContributesMultibinding
+import okhttp3.Interceptor
+import okhttp3.Response
+import timber.log.Timber
+import javax.inject.Inject
+
+@ContributesMultibinding(
+    scope = AppScope::class,
+    boundType = PixelInterceptorPlugin::class
+)
+class AtpUnprotectedAppsBucketPixelRemovalInterceptor @Inject constructor() : Interceptor, PixelInterceptorPlugin {
```

Comment:
Yes, the `ATB` is not needed, but it's removed by another interceptor because this pixel starts with `m_atp`, that's why I didn't do anything for this.
I'll rename the `PixelEmailRemovalInterceptor` and use it as it's not great to keep adding interceptors, especially since `atb` and `appVersion` are not needed in most cases.